package.json

```diff
@@ -21,7 +21,7 @@
   },
   "dependencies": {
     "ember-cli-babel": "^6.6.0",
-    "ember-getowner-polyfill": "^1.1.1"
+    "ember-getowner-polyfill": "^1.1.0 || ^2.0.0"
```

Comment:
This helps with deduplication in `node_modules` basically. If in your app you depend on ember-cookies and another addon then when ember-cookies depends on `"ember-getowner-polyfill": "^2.0.0"` and the other addon depends on `"ember-getowner-polyfill": "^1.1.0"` you need to install both `"^1.1.0"` and `"^2.0.0"`. As we know ember-cookies works with both "^1.1.0"` and "^2.0.0"` though we can state that and help make everyone's `node_modules` folders smaller.